src/viam/components/power_sensor/client.py

```diff
@@ -55,3 +57,11 @@ async def do_command(self, command: Mapping[str, ValueTypes], *, timeout: Option
         request = DoCommandRequest(name=self.name, command=dict_to_struct(command))
         response: DoCommandResponse = await self.client.DoCommand(request, timeout=timeout)
         return struct_to_dict(response.result)
+
+    async def get_geometries(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None) -> List[Geometry]:
```

Comment:
This was only added since it was failing tests because of sensor also having get_geometries. But this method doesn't exist in the proto file of PowerSensor, so I'm not entirely sure how this should go.

Reply:
can you implement it and raise an error in the PowerSensor parent class?